GCRCatalogs/photoz_magerr.py

```diff
@@ -182,7 +260,7 @@ def __getitem__(self, key):
     @property
     def handle(self):
         if self._handle is None:
-            try:
+            try: 
```

Comment:
Remove trailing spaces
```suggestion
            try:
```